src/selectors/Entities/name.js

```diff
@@ -87,3 +87,11 @@ export const selectVaccinePatientHistory = state => {
   const fullQuery = `(${inQuery}) AND ${baseQueryString} AND itemBatch.item.isVaccine == true`;
   return inQuery ? UIDatabase.objects('TransactionBatch').filtered(fullQuery).slice() : [];
 };
+
+export const selectWasPatientVaccinatedToday = state => {
+  const history = selectVaccinePatientHistory(state);
+
+  const currentDate = new Date();
+  return !!history.filter(historyRecord => historyRecord.confirmDate >= currentDate.getDate() - 1)
```

Comment:
`getDate()` returns day of the month!
I'm getting `30` for getDate, and the patient I'm using was vaccinated on 30th May, so it won't let me vaccinate today 😜

Reply:
Am also concerned that I won't be able to vaccinate today if someone was jabbed yesterday

Reply:
Haha.... oops. Better to just check current date is the same? (ignoring timestamp) I guess no one should be doing vaccinations at midnight or anything like that?

Reply:
Oh it's all wrong, I think I somehow left out the bit that checked the actual transaction too... as the history record itself doesn't even have a confirmDate 🤦

Reply:
The below is my suggestion :) 
Comparing the javascript version of unix time.. so we limit to vaccinations within the last 24hours.

along with this, of course:
`import { MILLISECONDS_PER_DAY } from '../../database/utilities/constants';`

Reply:
> as the history record itself doesn't even have a confirmDate 

I thought the same! and couldn't see why it was working, and getting a confirmDate! but you were right, both times :)